### ISSUE TITLE
Color contrast issue for orange links in 'action' section, div overlap issue

### ISSUE THREAD

**Issue by dandalpiaz** [web-illinois/ilw-footer#26]
The links in the 'action' section of the footer are being flagged by automated accessibility checkers for a color contrast issue:

> Element has insufficient color contrast of 3.04 (foreground color: #ffffff, background color: #ff5f05, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

It's super close to qualifying as 'large text' which would only require a 3.0:1 ratio, but I think the font weight of 600 doesn't technically qualify as bold. A font weight of 700 seems to pass okay, if that would be a viable change to the design.

<img width="955" height="471" alt="footer with action links" src="https://github.com/user-attachments/assets/91e3f29b-5f81-4bd9-8a3b-a52d8865a5c2" />

Also, I notice that in certain content situations in the footer, these links cannot be clicked or hovered with the mouse. It looks like the `<div class="content">` section may overlap the 'actions' section in some cases. Some quick example HTML below:

```html
<!DOCTYPE html> 
<html lang="en"> 
<head> 
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> 
    <meta name="viewport" content="width=device-width, initial-scale=1"> 
    <link rel="icon" href="https://cdn.brand.illinois.edu/favicon.ico"> 
    <link rel="dns-prefetch" href="https://cdn.toolkit.illinois.edu"> 
    <link rel="dns-prefetch" href="https://cdn.disability.illinois.edu"> 
    <link rel="stylesheet" href="https://cdn.toolkit.illinois.edu/3/toolkit.css">
    <script src="https://cdn.toolkit.illinois.edu/3/toolkit.js"></script>
</head>
<body style="margin: 0;">
<ilw-page>
    <ilw-header slot="header"></ilw-header>
    <main></main>
    <ilw-footer slot='footer'>
      <a slot="primary-unit" href="/">Toolkit Builder 3</a>
      <a slot="site-name" href="/sample_page/index.html">Whispering Pines College</a>
      <address slot="address">
        <div>1313 Pine Avenue</div>
        <div>Normal, ZZ 99999</div>
        <div>Phone: 555-874-3000</div>
        <div>Email: contact@iou.edu</div>
      </address>
      <div slot="actions">
        <a href="#">Give</a>
        <a href="#">Apply</a>
      </div>
    </ilw-footer>
</ilw-page>
</body>
</html>
```

**Comment by areynold** [comment]
Thanks Dan!

Which accessibility checker is flagging the contrast fails? Color contrast shows up as a pass for me (albeit borderline) at 600. AInspector calls 600 a large font and 700 both large and bold, both of which trigger the 3:1 rule.

Even if the current ratio technically passes, it's an easy change. It may be worth bumping the font weight up in order to avoid confusion.

Could you give more information about the conditions that cause link overlap? I wasn't able to block any of my clicks using the sample html.

**Comment by dandalpiaz** [comment]
Hi Andrew! I was using the Axe DevTools extension and the built-in Lighthouse tool in Chrome - I think they both use the axe-core ruleset. Probably their own interpretation of 'bold', which I could see varying a bit between different tools. 

For the overlap issue, I seemed to have been able to fix it by adding `type="module"` to the toolkit JS include:

```html
<script src="https://cdn.toolkit.illinois.edu/3/toolkit.js" type="module"></script>
```

If that's a best practice for using the toolkit, I could go ahead and make a pull request to get the starter code updated on the builder page, https://builder3.toolkit.illinois.edu/generic_html/index.html . 

Thanks!

**Comment by dandalpiaz** [comment]
Sorry, realized I forgot to provide additional context for the overlap I was seeing:

I tried that sample HTML on Chrome and Firefox on Windows and Chrome on macOS. When I don't have the `type="module"` in the JS include, I'll see this overlap of the 'content' div which prevents a click of the orange links/buttons at desktop sized viewports:

<img width="994" height="244" alt="screenshot of content overlap" src="https://github.com/user-attachments/assets/40d8274f-6b9a-4af7-a55d-102971ca79e6" />

It looks like a `section-grid--no-action` class is getting applied in a parent div when the JS isn't loaded as a module. Without module attribute:

```html
<div class="site section section-grid section-grid--no-action"> 
```

With module attribute:

```html
<div class="site section section-grid">
```
